manifests/03-cluster-role-binding.yaml

```diff
@@ -0,0 +1,6 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cluster-storage-operator
+  annotations:
+    release.openshift.io/delete: "true"
```

Comment:
Can we have these objects in a single file, say `1000-stale-cruft-whatever.yaml`?

Reply:
Yeah, I think that's a good idea, I kind of debated whether or not to do that. I'll make that change.